Convert legal PDFs to markdown

purpledrosophila opened the pull request:
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [ x] REQUIRED: Do you have an Issue **assigned to you** within the v1 milestone for this PR?  Put the Issue number here: 812
- [ ] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [ x] Tested your changes, especially after any code review iterations.
- [ x] Included any relevant screenshots of UI updates. (N/A)
- [x ] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x ] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x ] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

After all boxes above are checked, request and receive an Approved review from any team member knowledgable in the area (TODO team member list).  Once approved, the team member will assign your review to a Committer or use the `needs-merge` label.

## What does this PR accomplish?

Converts legal PDFs to Markdown as start of accessibility edits.

<!-- Title should be a short phrase, e.g. "Adds survey functionality". -->

<!-- Detailed description can include any design decisions you want reviewers to take note of. -->

This handles the simple conversion, but I have noticed some inconsistencies in phrasing and places where terms could be clarified. I would also suggest creating a legal terms glossary as part of the issue this request addresses. Running the text through Readable.org also showed that we could potentially simplify and clarify the text; I suggest we do that before closing this issue.

<!-- List all issue numbers affected and closed by this PR. -->

812 -- Recommend not closing this yet.

## Did you add any dependencies?

<!-- List each added dependency and justifications (see the Guidelines) -->

## How did you test the change?

Checked to make sure Markdown was valid.

<!-- If relevant, add any screenshots of your UI changes. -->

advayDev1 commented:
Thanks @purpledrosophila - we shouldn't change any of the wording except through advice by legal counsel.

Can you confirm this is an exact copy of the current PDFs?

purpledrosophila commented:
@advayDev1 Yes, I can confirm. The only thing I changed was a whitespace error that I caught while converting. Somebody forgot a space between a couple words, and I fixed it.

britannio commented on docs/TERMS.md:
```diff
@@ -0,0 +1,30 @@
+# WHO COVID-19 App Terms
+
+These App Terms govern your use of the service (“Services”) provided by the World Health Organization (‘WHO”, “we” “us” or “our”) via our mobile or web apps (“Apps”). By using our Service, you accept these App Terms. These App Terms supplement our Privacy Policy. We
```
```suggestion
These App Terms govern your use of the service (“Services”) provided by the World Health Organization (‘WHO”, “we” “us” or “our”) via our mobile or web apps (“Apps”). By using our Service, you accept these App Terms. These App Terms supplement our [Privacy Policy](https://whocoronavirus.org/privacy). We
```

purpledrosophila replied:
Fixed. :)

britannio commented on docs/PRIVACY.md:
```diff
@@ -0,0 +1,46 @@
+# WHO COVID-19 App Privacy Notice
+
+This App Privacy Notice supplements the general World Health Organization (“WHO” or “us”, “we” or “our”) Privacy Policy available at [https://www.who.int/about/who-we-are/privacy-policy][1] (the “WHO Privacy Policy”), and applies to mobile applications and services made available by the WHO (each an “App” or “Service”).
```
```suggestion
This App Privacy Notice supplements the general World Health Organization (“WHO” or “us”, “we” or “our”) Privacy Policy available at [https://www.who.int/about/who-we-are/privacy-policy](https://www.who.int/about/who-we-are/privacy-policy) (the “WHO Privacy Policy”), and applies to mobile applications and services made available by the WHO (each an “App” or “Service”).
```

britannio commented on docs/PRIVACY.md:
```diff
@@ -0,0 +1,46 @@
+
+Currently in connection with the WHO COVID-19 App we use the following third-party services and application stores. We may use additional services for testing and internal use.
+
+* Apple Store (iTunes Store) (Privacy Policy)
+* Google Analytics (Privacy Policy and Terms)
+* Google Cloud (Privacy Policy)
+* Google Firebase (Privacy Policy)
+* Google Play Store (Privacy Policy)
```
```suggestion
* Apple Store (iTunes Store) ([Privacy Policy](https://www.apple.com/legal/privacy/en-ww/))
* Google Analytics ([Privacy Policy](https://support.google.com/analytics/answer/6004245) and [Terms](https://www.google.com/analytics/terms/))
* Google Cloud ([Privacy Policy](https://cloud.google.com/security/privacy))
* Google Firebase ([Privacy Policy](https://firebase.google.com/support/privacy))
* Google Play Store ([Privacy Policy](https://policies.google.com/privacy?hl=en&gl=us))
```

purpledrosophila replied:
Fixed. Will commit in just a minute when I fix the rest of this page's requests.

britannio commented on docs/PRIVACY.md:
```diff
@@ -0,0 +1,46 @@
+
+## Third-Party Apps and Services
+
+Third-party applications and/or services linked to or embedded in the App governed by those third-party security and privacy policies (see third party tools). If you consent to receiving push notifications, a third-party provider may collect device and regional location information, but your identity or Personal Data will not be revealed to us as a result of enabling such notifications. You may s ...
```
The pdf has blue text for (see `third party tools`) here but no associated link

britannio replied:
Actually, I think it's supposed to refer to the heading at the bottom of the doc.

purpledrosophila replied:
Got it, will add that link now.

britannio commented on docs/PRIVACY.md:
```diff
@@ -0,0 +1,46 @@
+
+## Third-Party Apps and Services
+
+Third-party applications and/or services linked to or embedded in the App governed by those third-party security and privacy policies (see third party tools). If you consent to receiving push notifications, a third-party provider may collect device and regional location information, but your identity or Personal Data will not be revealed to us as a result of enabling such notifications. You may s ...
```
```suggestion
Third-party applications and/or services linked to or embedded in the App governed by those third-party security and privacy policies (see [third party tools](#current-third-party-tools)). If you consent to receiving push notifications, a third-party provider may collect device and regional location information, but your identity or Personal Data will not be revealed to us as a result of enabling such notifications. You may stop push notifications by adjusting your device settings.
```

purpledrosophila replied:
Fixed. :)

purpledrosophila commented:
All links should be updated now. Anything else I should tweak?

avkvirtru commented on docs/PRIVACY.md:
```diff
@@ -0,0 +1,53 @@
+
+The use of data described in the WHO Privacy Policy applies to the Service. Additionally, we use Usage Data to enable participation in interactive features of our Service and to provide you with information if you opt into receiving information through the Service or engage with interactive features.
+
+## Third-Party Apps and Services
```
Content mismatch: "Third Party" doesn't have a dash in the [current privacy.pdf](https://github.com/WorldHealthOrganization/app/blob/069bfe9/server/appengine/src/main/webapp/privacy.pdf)

avkvirtru commented on docs/PRIVACY.md:
```diff
@@ -0,0 +1,53 @@
+
+## Third-Party Apps and Services
+
+Third-party applications and/or services linked to or embedded in the App governed by those third-party security and privacy policies (see [third party tools][8]). If you consent to receiving push notifications, a third-party provider may collect device and regional location information, but your identity or Personal Data will not be revealed to us as a result of enabling such notifications. You  ...
```
Content mismatch: "third party provider" doesn't have a dash in the [current privacy.pdf](https://github.com/WorldHealthOrganization/app/blob/069bfe9/server/appengine/src/main/webapp/privacy.pdf)

avkvirtru commented on docs/PRIVACY.md:
```diff
@@ -0,0 +1,53 @@
+
+## Data Retention and Transfer
+
+We retain and use Personal Data and Usage Data only as long as is necessary for the purposes set out in our Privacy Policy and this Privacy Notice, or to the extent necessary to comply with legal obligations, resolve disputes, or enforce legal agreements and policies. We transfer the data, including Personal Data, to the U.S.A. for processing and storage, and by using or submitting information vi ...
```
Content mismatch: "Useage Data" in the [current privacy.pdf](https://github.com/WorldHealthOrganization/app/blob/069bfe9/server/appengine/src/main/webapp/privacy.pdf)

The original word "useage" does [appear to be a misspelling](https://linguistlist.org/ask-ling/message-details1.cfm?asklingid=200312267) in American and British English. But I would confirm even typos with legal…

purpledrosophila commented:
OK; if correcting typos like that is necessary to confirm with legal, then
I would suggest doing a full rewrite for the sake of readability, then
looping legal in on this. I didn't think that simply correcting typos would
require the legal department's approval -- the terms PDF listed
"third-party" as one word with a hyphen, so I assumed that any place where
it was 2 words was a simple typo. As I've mentioned previously, there are
parts of the current PDFs that could be quite difficult to understand
because of how the sentences are structured, and I'm certain we could
simplify them without losing meaning. Readability is one of my specialties,
and as I'm sure you know, that goes along with accessibility. Who would we
have to talk to in legal about this?

On Mon, Apr 6, 2020 at 12:45 PM Arthur Klepchukov <notifications@github.com>
wrote:

> *@avkvirtru* commented on this pull request.
>
> There are minor content mismatches going from PDF to Markdown: 2 dashes
> inserted and a corrected spelling. Not sure who should decide if these
> should change…
> ------------------------------
>
> In docs/PRIVACY.md
> <https://github.com/WorldHealthOrganization/app/pull/868#discussion_r404232473>
> :
>
> > +By using a Service, you agree to the collection and use of data in accordance with these policies. We do not disclose, share, sell, or transfer any information about Service users unless required by law or for law enforcement.
>
> +
>
> +## Data We Collect
>
> +
>
> +When you use our Service we:
>
> +
>
> +* Do not collect data that identifies you personally (“Personal Data”) unless you choose to provide that information to us.
>
> +* Do collect information that does not identify you personally, including data collected automatically either generated by the use of the Service or from the Service infrastructure (“Usage Data”). Usage Data may include location information, which we may use to provide you with more relevant information or language choices.
>
> +
>
> +Our Apps are not intended to be used by or to solicit information of any kind from children under age 13.
>
> +
>
> +## How We Use Data
>
> +
>
> +The use of data described in the WHO Privacy Policy applies to the Service. Additionally, we use Usage Data to enable participation in interactive features of our Service and to provide you with information if you opt into receiving information through the Service or engage with interactive features.
>
> +
>
> +## Third-Party Apps and Services
>
>
> Content mismatch: "Third Party" doesn't have a dash in the current
> privacy.pdf
> <https://github.com/WorldHealthOrganization/app/blob/069bfe9/server/appengine/src/main/webapp/privacy.pdf>
> ------------------------------
>
> In docs/PRIVACY.md
> <https://github.com/WorldHealthOrganization/app/pull/868#discussion_r404233175>
> :
>
> > +## Data We Collect
>
> +
>
> +When you use our Service we:
>
> +
>
> +* Do not collect data that identifies you personally (“Personal Data”) unless you choose to provide that information to us.
>
> +* Do collect information that does not identify you personally, including data collected automatically either generated by the use of the Service or from the Service infrastructure (“Usage Data”). Usage Data may include location information, which we may use to provide you with more relevant information or language choices.
>
> +
>
> +Our Apps are not intended to be used by or to solicit information of any kind from children under age 13.
>
> +
>
> +## How We Use Data
>
> +
>
> +The use of data described in the WHO Privacy Policy applies to the Service. Additionally, we use Usage Data to enable participation in interactive features of our Service and to provide you with information if you opt into receiving information through the Service or engage with interactive features.
>
> +
>
> +## Third-Party Apps and Services
>
> +
>
> +Third-party applications and/or services linked to or embedded in the App governed by those third-party security and privacy policies (see [third party tools][8]). If you consent to receiving push notifications, a third-party provider may collect device and regional location information, but your identity or Personal Data will not be revealed to us as a result of enabling such notifications. You may stop push notifications by adjusting your device settings.
>
>
> Content mismatch: "third party provider" doesn't have a dash in the current
> privacy.pdf
> <https://github.com/WorldHealthOrganization/app/blob/069bfe9/server/appengine/src/main/webapp/privacy.pdf>
> ------------------------------
>
> In docs/PRIVACY.md
> <https://github.com/WorldHealthOrganization/app/pull/868#discussion_r404235610>
> :
>
> > +
>
> +## How We Use Data
>
> +
>
> +The use of data described in the WHO Privacy Policy applies to the Service. Additionally, we use Usage Data to enable participation in interactive features of our Service and to provide you with information if you opt into receiving information through the Service or engage with interactive features.
>
> +
>
> +## Third-Party Apps and Services
>
> +
>
> +Third-party applications and/or services linked to or embedded in the App governed by those third-party security and privacy policies (see [third party tools][8]). If you consent to receiving push notifications, a third-party provider may collect device and regional location information, but your identity or Personal Data will not be revealed to us as a result of enabling such notifications. You may stop push notifications by adjusting your device settings.
>
> +
>
> +## Disclaimer
>
> +
>
> +THE MATERIALS EMBODIED IN THIS SERVICE ARE PROVIDED “AS-IS” AND WITHOUT WARRANTY OF ANY KIND, EXPRESSED, IMPLIED OR OTHERWISE, INCLUDING WITHOUT LIMITATION, ANY WARRANTY OF FITNESS FOR A PARTICULAR PURPOSE. IN NO EVENT SHALL THE WORLD HEALTH ORGANIZATION BE LIABLE TO YOU OR ANYONE ELSE FOR ANY DIRECT, SPECIAL, INCIDENTAL, INDIRECT OR CONSEQUENTIAL DAMAGES OF ANY KIND, OR ANY DAMAGES WHATSOEVER, WHETHER OR NOT ADVISED OF THE POSSIBILITY OF SUCH LOSS, HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, ARISING OUT OF OR IN CONNECTION WITH THE POSSESSION, USE OR PERFORMANCE OF THIS SERVICE.
>
> +
>
> +## Data Retention and Transfer
>
> +
>
> +We retain and use Personal Data and Usage Data only as long as is necessary for the purposes set out in our Privacy Policy and this Privacy Notice, or to the extent necessary to comply with legal obligations, resolve disputes, or enforce legal agreements and policies. We transfer the data, including Personal Data, to the U.S.A. for processing and storage, and by using or submitting information via the Service you consent to such transfer.
>
>
> Content mismatch: "Useage Data" in the current privacy.pdf
> <https://github.com/WorldHealthOrganization/app/blob/069bfe9/server/appengine/src/main/webapp/privacy.pdf>
>
> The original word "useage" does appear to be a misspelling
> <https://linguistlist.org/ask-ling/message-details1.cfm?asklingid=200312267>
> in American and British English. But I would confirm even typos with legal…
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/WorldHealthOrganization/app/pull/868#pullrequestreview-388426622>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANENNIYUZLEUT3EJIG7H24LRLIBJDANCNFSM4L67PRNA>
> .
>

avkvirtru commented:
@purpledrosophila @britannio I'm surprised this PR doesn't remove the server-side PDFs or the routing to them:

```java
    serve("/terms").with(forwardTo("/terms.pdf"));
    serve("/privacy").with(forwardTo("/privacy.pdf"));
```
(from https://github.com/WorldHealthOrganization/app/blob/master/server/appengine/src/main/java/who/WhoServletModule.java#L25)

Also seems like we'd have to serve HTML, not Markdown at those URLs.

I can't test the accessibility of the content in the app without being able to navigate to these new terms and privacy policy from within the app.

britannio commented:
@avkvirtru I was completely oblivious to those files. If we switched from PDFs to HTML then the link in the app would show a website instead of downloading a PDF. The other option was converting the PDFs to markdown and displaying the markdown in the app, that way it's also version controlled. 

Maybe now is the time to decide what should actually be done, from a legal side are there any rks of showing a static TOS & PP in the event that we update them?

purpledrosophila commented:
I would suggest writing the files as HTML. They shouldn't be that hard to
maintain; they're pretty simple. Is there a unified HTML template for the
entire app? If so, I can check it for accessibility and use that. Or, I
could write a new one.

On Mon, Apr 6, 2020 at 1:04 PM Britannio Jarrett <notifications@github.com>
wrote:

> @avkvirtru <https://github.com/avkvirtru> I was completely oblivious to
> those files. If we switched from PDFs to HTML then the link in the app
> would show a website instead of downloading a PDF. The other option was
> converting the PDFs to markdown and displaying the markdown in the app,
> that way it's also version controlled.
>
> Maybe now is the time to decide what should actually be done, from a legal
> side are there any rks of showing a static TOS & PP in the event that we
> update them?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/WorldHealthOrganization/app/pull/868#issuecomment-609918080>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANENNI3NOYVSMAIO4SFQQ63RLIDQFANCNFSM4L67PRNA>
> .
>

avkvirtru commented:
> @avkvirtru I was completely oblivious to those files. If we switched from PDFs to HTML then the link in the app would show a website instead of downloading a PDF. The other option was converting the PDFs to markdown and displaying the markdown in the app, that way it's also version controlled.

@britannio the current approach is to show a web view in the app that renders those PDFs. HTML makes the most sense to me.

advayDev1 commented:
First - thank you!

Can you make sure these are in a docs/legal folder?

We can then add some rules to make sure they are reviewed by legal before they get committed unlike all other parts of the codebase.

purpledrosophila commented:
You're welcome. 🙂 Just to be clear, do you want me to put the current
Markdown I created in a docs/legal folder, or do you want me to convert
them into HTML before making that folder? Also, are there any other legal
docs I should convert and put in there while I'm at it?

On Mon, Apr 6, 2020, 15:11 Advay Mengle <notifications@github.com> wrote:

> First - thank you!
>
> Can you make sure these are in a docs/legal folder?
>
> We can then add some rules to make sure they are reviewed by legal before
> they get committed unlike all other parts of the codebase.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/WorldHealthOrganization/app/pull/868#issuecomment-609982932>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANENNI4ZQQBOL3WCFGOJ7MLRLISMPANCNFSM4L67PRNA>
> .
>

advayDev1 commented:
Every document needing legal counsel review can be in docs/legal

britannio commented:
> Is that everything currently in the docs folder, then?

I believe you need to create a `legal` folder under docs and place the markdown files there.
i.e. `docs/legal/TERMS.md` instead of `docs/TERMS.md`

purpledrosophila commented:
I understand that, but do I have to put everything currently in the docs
folder under docs/legal? And, should they be HTML instead of Markdown,
based on our conversations on the pull request?

On Thu, Apr 9, 2020, 15:39 Britannio Jarrett <notifications@github.com>
wrote:

> Is that everything currently in the docs folder, then?
>
> I believe you need to create a legal folder under docs and place the
> markdown files there.
> i.e. docs/legal/TERMS.md instead of docs/TERMS.md
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/WorldHealthOrganization/app/pull/868#issuecomment-611716519>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANENNI6U2YW2TN57DLYLMZLRLYQABANCNFSM4L67PRNA>
> .
>

britannio commented:
> I understand that, but do I have to put everything currently in the docs folder under docs/legal? And, should they be HTML instead of Markdown, based on our conversations on the pull request?

It should just be the terms and privacy files. My guess is that markdown is preferred as we can just copy the `docs/legal` stuff into the app and use something like [flutter_markdown](https://pub.dev/packages/flutter_markdown) to display it, I don't think it'll cause accessibility issues. Maybe the design team will need to get involved.

purpledrosophila commented:
Got it, I will do that today.

On Thu, Apr 9, 2020, 16:05 Britannio Jarrett <notifications@github.com>
wrote:

> I understand that, but do I have to put everything currently in the docs
> folder under docs/legal? And, should they be HTML instead of Markdown,
> based on our conversations on the pull request?
>
> It should just be the terms and privacy files. My guess is that markdown
> is preferred as we can just copy the docs/legal stuff into the app and
> use something like flutter_markdown
> <https://pub.dev/packages/flutter_markdown> to display it, I don't think
> it'll cause accessibility issues. Maybe the design team will need to get
> involved.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/WorldHealthOrganization/app/pull/868#issuecomment-611727378>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANENNI3WPENEERNVKXCQKH3RLYTBFANCNFSM4L67PRNA>
> .
>

epicfaace commented:
> We can then add some rules to make sure they are reviewed by legal before they get committed unlike all other parts of the codebase.

@advayDev1 Maybe we can use CODEOWNERS to require review from you / @hspinks (or the lawyers, do they have GitHub accounts?) for the `docs/legal` directory?

advayDev1 commented:
> > We can then add some rules to make sure they are reviewed by legal before they get committed unlike all other parts of the codebase.
> 
> @advayDev1 Maybe we can use CODEOWNERS to require review from you / @hspinks (or the lawyers, do they have GitHub accounts?) for the `docs/legal` directory?

CODEOWNERS already requires that only those of us who are admins can approve docs changes.

Sorry for the delay @purpledrosophila - waiting on an internal approval here.

advayDev1 commented:
@purpledrosophila alright we’ve got the go ahead - just need to check that the pdfs and markdown match.  Can you update this branch with respect to master?

purpledrosophila commented:
Can you please give me the links to the most recent PDFs? Just want to make
sure I'm comparing the right files, didn't want to assume the old links
were still current.

On Fri, Apr 17, 2020, 11:19 Advay Mengle <notifications@github.com> wrote:

> *@advayDev1* requested changes on this pull request.
>
> Please confirm all differences between most recent pdfs and master
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/WorldHealthOrganization/app/pull/868#pullrequestreview-395546569>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANENNIZ3UHMVGGXUUST4LA3RNBXQNANCNFSM4L67PRNA>
> .
>

purpledrosophila commented:
OK, something weird is going on with my GitHub setup, and I'm not sure what it is. I'm trying to work on another issue, can't seem to do it, and can't seem to update my repository, for some reason. I've been trying to figure out what's going on, not having much success. If I delete my fork and the branch attached to this pull request, will we have to create a new issue for this? I'm sorry about this issue; I'm really not sure what's going on. I've got the updated Markdown, though; I just need the links to the PDFs again to make sure that they're the same.

purpledrosophila commented:
OK, sorry for the spamming, I think I somehow managed to update my branch properly. Once I have the old PDFs in hand to double-check one more time, we should be good.

advayDev1 commented:
@purpledrosophila https://whoapp.org/privacy https://whoapp.org/terms

advayDev1 commented:
@purpledrosophila how's this going

purpledrosophila commented:
Sorry for the delay, I've been sick. Not COVID, thank goodness. I was just
going to look at this now, I'll have it committed in a few minutes.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Fri, Apr 24, 2020 at 7:59 PM Advay Mengle <notifications@github.com>
wrote:

> @purpledrosophila <https://github.com/purpledrosophila> how's this going
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/WorldHealthOrganization/app/pull/868#issuecomment-619285247>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANENNI6UH4QHYUKXXBDHWRLROIRURANCNFSM4L67PRNA>
> .
>

purpledrosophila commented:
Hi all,

I fixed a few typos in PRIVACY; other than that, the Markdown is an exact match to the PDFs. The Third Party Tools section looks a little different in the Markdown, but that was by request; if you want me to change it back let me know.

I noticed a few more things we may want to check with legal for the sake of clarity. Note that I did not make these changes yet, I'm just suggesting them. As far as I can tell, the content of the files is good, no more changes are essential.

-----

Under "Prohibited Uses" in TERMS, the PDF says:

"You agree not to use the App in any way that is unlawful, illegal or unauthorised; advocates, promotes or assists any unlawful; or is likely to disrupt the Service."

Shouldn't it say:

"You agree not to use the App in any way that is unlawful, illegal or unauthorised; advocates, promotes or assists any unlawful activities; or is likely to disrupt the Service."

Seems like a word might have been missed there?

-----

Also, in terms of clarity -- TERMS says this:

"THE APP IS IN NO WAY TO BE CONSIDERED OR USED AS A REPLACEMENT FOR PERSONAL PROFESSIONAL MEDICAL ADVICE FROM YOUR DOCTOR."

Wouldn't it be clearer to say:

"THE APP IS IN NO WAY TO BE CONSIDERED OR USED AS A REPLACEMENT FOR PROFESSIONAL MEDICAL ADVICE FROM YOUR PERSONAL DOCTOR."

"Personal professional" seems like it could be confusing.

-----

Lastly, just a thought from me -- I ran TERMS and PRIVACY through a readability analyzer, and they have higher reading levels than they probably should if we want people to understand them. PRIVACY has a reading level of 9.8, which isn't that bad, but an ideal would be 8 or lower, as in an 8th grade reading level. TERMS has a reading level of over 13, which means no one hasn't completed at least some college could read it easily. I'm aware that things have to be phrased in "legalese" to some degree, but is there any way we could simplify them a bit? I figure that the more understandable these are, the better off WHO will be.

advayDev1 commented:
@purpledrosophila hope you're feeling better and thank you!  

@stephliu I'll get the current version in - can you sync with counsel on the suggestions @purpledrosophila made?

advayDev1 commented:
@purpledrosophila what typos did you change?  `third-party` should keep its hyphen when used as an adjective

purpledrosophila commented:
Ah, I thought that third party was always 2 words. There were instances in
both documents where there it was sometimes written as 2 words, sometimes
written as one hyphenated word. Did I second-guess myself too much; should
they all have been one hyphenated word?

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Fri, Apr 24, 2020 at 11:22 PM Advay Mengle <notifications@github.com>
wrote:

> @purpledrosophila <https://github.com/purpledrosophila> what typos did
> you change? third-party should keep its hyphen when used as an adjective
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/WorldHealthOrganization/app/pull/868#issuecomment-619312114>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANENNI2YHQ437CZ3GHEGHL3ROJJRDANCNFSM4L67PRNA>
> .
>

advayDev1 commented:
`third party` as a noun and `third-party` as an adjective.  Let's go back to whatever counsel had, and Stephanie will coordinate the on-going changes with counsel.

purpledrosophila commented:
Hmm, I learned something today. :) I will fix this right now.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Sat, Apr 25, 2020 at 2:30 PM Advay Mengle <notifications@github.com>
wrote:

> third party as a noun and third-party as an adjective. Let's go back to
> whatever counsel had, and Stephanie will coordinate the on-going changes
> with counsel.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/WorldHealthOrganization/app/pull/868#issuecomment-619421989>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANENNI3EJSLANYEE72T6RXLROMT3NANCNFSM4L67PRNA>
> .
>

purpledrosophila commented:
@advayDev1 Do you want me to create another issue for the text of these documents?

advayDev1 commented:
Sure go for it @purpledrosophila

Chheomtit1993 reviewed (commented):
Chheomtit1993![Uploading 93710574...]
